### PR TITLE
feat: open source landing page

### PR DESCRIPTION
**Pull request by webrdaniel**

### CODE REVIEW

**Review by hanatsai — APPROVED** [review]
Thanks! it's beautifully done!

**Review by jbartadev — APPROVED** [review]
All good! 💯

**Review by HonzaTuron — APPROVED** [review]
Ship itttttt 🚀

**Review by B4nan — APPROVED** [review]
gave this a quick look as well, i dont see anything blocking. you can ignore the failed lychee check, it will always complain about links to new pages.

File: src/pages/open-source/index.tsx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                        </div>
+                    </Section>
+                    <Section
+                        heading="Actor templates"
+                        description="Actor templates help you quickly set up your web scraping projects.
+                            Save development time and get immediate access to all the features that the Apify platform has to offer."
+                        headingAs="h2"
+                    >
+                        <div className="actorTemplates">
+                            <ActorTemplates displayedTemplatesIds={['python-start', 'js-start', 'ts-start']} />
```
Comment (baldasseva) [inline src/pages/open-source/index.tsx:224]: Nit: If you want, we have a better icon for dark theme for HTTPX here: https://github.com/apify/apify-core/blob/develop/src/console/frontend/public/img/logos_other/logo_httpx.svg

**Review by baldasseva — APPROVED** [review]

**Pull request merged**